src/WormholeConstantFee.sol

```diff
@@ -21,12 +21,28 @@ import {WormholeGUID} from "./WormholeGUID.sol";
 
 contract WormholeConstantFee is WormholeFees {
     uint256 immutable public fee;
+    uint256 immutable public ttl;
 
-    constructor(uint256 _fee) {
+    /**
+    * @param _fee Constant fee in WAD
+    * @param _ttl Time in seconds to finalize flush (not wormhole). Usually, for optimistic rollups should be 8 days.
```

Comment:
We need to run an actual calculus for worst-case time here, so I don't think we should be saying 8 days for optimistic rollups.

Reply:
I decided to put a specific time here as a sane default. 8 days as I assume that flush will happen at worst 1 day after wormhole was created and it will take 7 days to finalize it. Should I delete it?